### DISCUSSION TITLE
fan noisy increase after change from laptop mode to stand mode.

### DISCUSSION THREAD

**Discussion by xiyivan** [seerge/g-helper#1252] [Q&A]
The settings in the G-helper are identical, but the fans just bump from 10% to 50% at the same temperature. I don't think the computer is follwing the fan curve I set and is using its default settings.

**Comment by seerge** [comment]
@xiyivan is it for x13 ? i think someone on reddit was also mentioning such behaviour.

It’s how bios is programmed i guess
